resources/views/admin/log.twig

```diff
@@ -75,7 +75,13 @@
                             {%- endif %}
 
                             <tr>
-                                <td class="table-{{ td_type }} text-nowrap">{{ entry.created_at.format(__('general.datetime')) }}</td>
+                                <td class="table-{{ td_type }} text-nowrap">
+                                    <span id="message-{{ entry.id }}" class="ref-id"></span>
+                                    {{ entry.created_at.format(__('general.datetime')) }}
+                                    <small class="text-muted">
+                                        <a class="ref-link" href="#message-{{ entry.id }}">{{ m.icon('link') }}</a>
```

Comment:
When I click on the link of this log message:
<img width="1492" height="47" alt="Screenshot 2025-12-17 at 16 32 16" src="https://github.com/user-attachments/assets/195ec02a-bd22-49d6-854b-65162256ab42" />

My page looks like this:
<img width="1512" height="848" alt="Screenshot 2025-12-17 at 16 32 07" src="https://github.com/user-attachments/assets/e8b8cb7a-e881-4934-ad7c-8ecf131a7148" />

Reply:
Also I would prefer a column for the link (with no header title to keep it narrow)

Reply:
Moved to  #1593